jbmc/regression/Makefile

```diff
@@ -8,7 +8,7 @@ DIRS = janalzyer-taint \
        jbmc-inheritance \
        jbmc-strings \
        jdiff \
-       string-smoke-tests \
+       strings-smoke-tests \
```

Comment:
I'd suggest to remove lines 15-18 in this Makefile as they 1) are unnecessary and 2) removing them would have made this problem surface earlier.